payload.tex

```diff
@@ -379,6 +379,85 @@ \chapter{Trace Encoder Output Packets} \label{packets}
   \end{tabulary}
 \end{table}
 
+\begin{table}[htp]
+  \centering
+  \caption{Packet Payload Format 0, subformat 0 - jump target index, branch map}
+  \label{tab:te_inst0-0-cache-map}
+  \begin{tabulary}{\textwidth}{|l|p{35mm}|p{80mm}|}
+    \hline
+    {\bf Field name} & {\bf Bits} & {\bf Description} \\
+    \hline
+    \textbf{format}	& 2	& 00 (optional): formats for optional efficiency improvements\\

```

Comment:
I'm confused about "optional" which I initially believe means that this field is optionally included.  But that can't be true or else all the other fields might shift by two bits and there's no way to deterministically parse this.

Reply:
Yes, I can see that.  I didn't mean that the field was optional - optional is the enumeration I chose for format 0, as it is used for packet formats for optional extensions.  I'll rename.